### PR TITLE
Update javadoc ant to include runners/ and exclude modules with a wildcard

### PR DESCRIPTION
**Pull request by aaltay**
R: @davorbonaci 

(cc: @sb2nov)

### CODE REVIEW

File: sdks/java/javadoc/ant.xml [hunk 1]
```diff
@@ -70,19 +81,15 @@
...
-      <excludepackage name="org.apache.beam.examples.complete.game"/>
-      <excludepackage name="org.apache.beam.examples.complete.game.injector"/>
-      <excludepackage name="org.apache.beam.examples.complete.game.utils"/>
-      <excludepackage name="org.apache.beam.runners.apex.translation"/>
-      <excludepackage name="org.apache.beam.runners.dataflow.internal"/>
-      <excludepackage name="org.apache.beam.runners.flink.examples"/>
-      <excludepackage name="org.apache.beam.runners.flink.translation"/>
-      <excludepackage name="org.apache.beam.runners.spark.examples"/>
-      <excludepackage name="org.apache.beam.runners.spark.translation"/>
+      <excludepackage name="org.apache.beam.examples.*"/>
```
Comment (dhalperi) [inline sdks/java/javadoc/ant.xml]: confirming differences from orig: you believe this works recursively? I think the original author did not.

File: sdks/java/javadoc/ant.xml [hunk 2]
```diff
@@ -70,19 +81,15 @@
...
+      <excludepackage name="org.apache.beam.fn.*"/>
```
Comment (dhalperi) [inline sdks/java/javadoc/ant.xml]: confirming diff from left -- exclude fn deliberately. (agreed).

File: sdks/java/javadoc/ant.xml [hunk 3]
```diff
@@ -70,19 +81,15 @@
...
+      <excludepackage name="org.apache.beam.runners.apex.translation.*"/>
+      <excludepackage name="org.apache.beam.runners.core.*"/>
+      <excludepackage name="org.apache.beam.runners.dataflow.internal.*"/>
+      <excludepackage name="org.apache.beam.runners.flink.examples.*"/>
+      <excludepackage name="org.apache.beam.runners.flink.translation.*"/>
+      <excludepackage name="org.apache.beam.runners.spark.examples.*"/>
+      <excludepackage name="org.apache.beam.runners.spark.translation.*"/>
       <excludepackage name="org.openjdk.jmh.infra.generated"/>
```
Comment (dhalperi) [inline sdks/java/javadoc/ant.xml]: drop this as the package no longer exists.

File: sdks/java/javadoc/ant.xml [hunk 4]
```diff
@@ -35,6 +35,17 @@
            results in one Java source tree. -->
       <mapper type="regexp" from="^.*/src/main/java/(.*)$$" to="\1"/>
     </copy>
+
+    <copy todir="${collect.dir}">
+      <fileset dir="../../../runners">
+        <include name="**/src/main/java/**/*.java"/>
+        <exclude name="**/maven-archetypes/**"/>
```
Comment (dhalperi) [inline sdks/java/javadoc/ant.xml]: drop this line -- there are no `maven-archetypes` inside of runners i believe.

**Comment by dhalperi** [comment]
R: @dhalperi

LGTM

File: sdks/java/javadoc/ant.xml (hunk 3 above)
Comment (aaltay) [inline sdks/java/javadoc/ant.xml]: Done

File: sdks/java/javadoc/ant.xml (hunk 2 above)
Comment (aaltay) [inline sdks/java/javadoc/ant.xml]: ack.

File: sdks/java/javadoc/ant.xml (hunk 1 above)
Comment (aaltay) [inline sdks/java/javadoc/ant.xml]: `excludepackage` can take a wildcard (see: https://ant.apache.org/manual/Tasks/javadoc.html)

File: sdks/java/javadoc/ant.xml (hunk 4 above)
Comment (aaltay) [inline sdks/java/javadoc/ant.xml]: Done.

**Pull request closed without merging**